### DISCUSSION TITLE
Monitor cpu usage on ltsp clients

### DISCUSSION THREAD

**Discussion by ikki0123** [ltsp/ltsp#828] [Q&A] [answered]
Hey guys.,
I don't know if I'm in the right place.
My scenario is as follows.
I'm using LTSP to run a cluster, it's running fine.
I would like to know if there is any way to monitor the CPU usage of my clients. I'm currently using Epoptes to visualize.

Is there another way to monitor these clients?

**Comment by alkisg** [comment, marked as answer]
Hi @ikki0123, on github, "issues" means that you want to report a bug in the project code.
While for general help, you need "discussions", there: https://github.com/ltsp/ltsp/discussions
I'll convert this issue into a discussion.

Regarding your query, it isn't specific to LTSP. You could use some network monitoring system: https://en.wikipedia.org/wiki/Comparison_of_network_monitoring_systems

**Reply by ikki0123** [reply]
Thanks!Do you have any recommendation?

**Reply by trendzetter** [reply]
I'm using zabbix for monitoring the health of hosts (not specific to ltsp)

**Reply by danboid** [reply]
I use Zabbix too. I'd recommend using their docker container because I had no luck upgrading my non-docker based Zabbix server.
